packages/cadl-ranch-specs/http/models/property-optional/mockapi.ts

```diff
@@ -55,36 +65,42 @@ function createMockApis(route: string, value: any): MockApiGetPut {
 const stringMock = createMockApis("string", "hello");
 Scenarios.Models_Property_Optional_String_getAll = passOnSuccess(stringMock.getAll);
 Scenarios.Models_Property_Optional_String_getDefault = passOnSuccess(stringMock.getDefault);
+Scenarios.Models_Property_Optional_String_getNull = passOnSuccess(stringMock.getNull);
```

Comment:
`null` is not the same as optional optional, this is technically an unexpected value. what are we trying to test here. 

This is how you would represent nullable in cadl(which will also be discouraged as optional is really what you want).
```cadl
model Foo {
  isNullable: string | null;
}
```

Reply:
Thanks @timotheeguerin for the detail knowledge!  I see so we don't need this for `property-optional`.

Though the `string|null` is discouraged, shall we add a case for nullable test in `property-types`?